docs/literate/src/files/hohqmesh_literate.jl

```diff
@@ -0,0 +1,521 @@
+#src # Unstructured meshes with HOHQMesh.jl
+
+# Trixi.jl supports numerical approximations on unstructured quadrilateral meshes
+# with the [`UnstructuredMesh2D`](@ref) mesh type.
+
+# The purpose of this tutorial is to demonstrate how to use the `UnstructuredMesh2D`
+# functionality of Trixi.jl. This begins by running and visualizing an available unstructured
+# quadrilateral mesh example. Then, the tutorial will demonstrate how to
+# conceptualize a problem with curved boundaries, generate
+# a curvilinear mesh using the available software in the Trixi.jl ecosystem,
+# and then run a simulation using Trixi.jl on said mesh.
+
+# Unstructured quadrilateral meshes can be made
+# with the [High-Order Hex-Quad Mesh (HOHQMesh) generator](https://github.com/trixi-framework/HOHQMesh)
+# created and developed by David Kopriva.
+# HOHQMesh is a mesh generator specifically designed for spectral element methods.
+# It provides high-order boundary curve information (needed to accurately set boundary conditions)
+# and elements can be larger (due to the high accuracy of the spatial approximation)
+# compared to traditional finite element mesh generators.
+# For more information about the design and features of HOHQMesh one can refer to its
+# [official documentation](https://trixi-framework.github.io/HOHQMesh/).
+
+# HOHQMesh is incorporated into the Trixi.jl framework via the registered Julia package
+# [HOHQMesh.jl](https://github.com/trixi-framework/HOHQMesh.jl).
+# This package provides a Julia wrapper for the HOHQMesh generator that allows users to easily create mesh files without the need to build
+# HOHQMesh from source. To install the HOHQMesh package execute
+# ```julia
+# import Pkg; Pkg.add("HOHQMesh")
+# ```
+# Now we are ready to generate an unstructured quadrilateral mesh that can be used by Trixi.jl.
+
+# ## Running and visualizing an unstructured simulation
+
+# Trixi.jl supports solving hyperbolic problems on several mesh types.
+# There is a default example for this mesh type that can be executed by
+
+using Trixi
+redirect_stdout(devnull) do # code that prints annoying stuff we don't want to see here #hide #md
+trixi_include(default_example_unstructured())
+end #hide #md
```

Comment:
A few times I added this code around `include`s, `solve`s or `summary_callback`s such that the output is not printed.
In the documentation and notebook, this will only appear with `trixi_include(default_example_unstructured())` etc.